vue/components/ui/molecules/upload-area/upload-area.vue

```diff
@@ -121,90 +130,33 @@ export const UploadArea = {
         disabled: {
             type: Boolean,
             default: false
-        },
-        draggingIcon: {
-            type: String,
-            default: null
-        },
-        files: {
-            type: Array,
-            default: () => []
-        },
-        multiple: {
-            type: Boolean,
-            default: true
-        },
-        accept: {
-            type: String,
-            default: null
-        }
-    },
-    computed: {
-        descriptionText() {
-            return this.dragging ? this.descriptionDragging : this.description;
-        },
-        icon() {
-            return this.draggingIcon ? this.draggingIcon : "cloud-upload";
-        },
-        classes() {
-            const base = { dragging: this.dragging, disabled: this.draggingDisabled };
-            return base;
         }
     },
     data: function() {
         return {
-            filesData: this.files,
             dragging: false,
-            draggingDisabled: false,
-            dragEnterTarget: null
+            fileInputRef: null
         };
     },
-    watch: {
-        files(value) {
-            if (!value || value.length === 0) this.clear();
-            this.filesData = value;
-        },
-        dragging(value) {
-            this.$emit("update:dragging", value);
-        }
+    mounted: function() {
+        this.fileInputRef = this.$refs.uploadButton?.$refs?.filesInput;

```

Comment:
Create a `initUploadArea` method in the mixin that does this (setting the `fileInputRef` based on the provided argument), so that you encapsulate the logic there.